dangerfile.js

```diff
@@ -0,0 +1,32 @@
+/* global danger, fail, message */
```

Comment:
You can remove the eslint ignore lines below and add a single `/* eslint-disable no-console */` comment at the top instead 🛩

Reply:
Done

Reply:
The comments are still in the code below?

Reply:
🤦‍♂️ Should be there now